src/main/java/quotes/App.java

```diff
@@ -37,6 +38,30 @@ public static String  randomQuote(Quote[] quotes) {
         return quotes[(int)(Math.random()*quotes.length)].quoteAndAuthorString();
     }
 
+    public static void webRequest(){
+        URL url = null;
+        try {
+            url = new URL("http://swquotesapi.digitaljedi.dk/api/SWQuote/RandomStarWarsQuote");
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
+        }
+        HttpURLConnection con = null;
+        try {
+            con = (HttpURLConnection) url.openConnection();
+            con.setRequestMethod("GET");
+            BufferedReader in = new BufferedReader(
+                    new InputStreamReader(con.getInputStream()));
+            String inputLine;
+            StringBuffer content = new StringBuffer();
+            while ((inputLine = in.readLine()) != null) {
+                content.append(inputLine);
+            }
+            in.close();
+            System.out.println(content.toString());
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Instead of throwing the error, you can have it read the offline quote;
`            Quote[] quotes = readFile();
            String quoteString = randomQuote(quotes);
            System.out.println(quoteString);`
But it is a bit neater if instead of catching it here, you have this method throw the IOException, and catch it with a try/catch in the main method.